repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/attributemappingvalidation/AttributeMappingPolicyValidationFilterTest.groovy

```diff
@@ -83,7 +83,7 @@ class AttributeMappingPolicyValidationFilterTest extends ReposeValveTest {
         mc.receivedResponse.code.toInteger() == SC_UNSUPPORTED_MEDIA_TYPE
 
         where:
-        contentType << [TEXT_PLAIN]
+        contentType << [TEXT_XML, APPLICATION_ATOM_XML, TEXT_PLAIN, APPLICATION_JSON]
```

Comment:
I guess I will wait to see if these pass, but I wouldn't expect them to? The `415` coming out of this filter should only be sent in the case where we know we don't accept the `Content-Type` (e.g., `text/plain`) but not for the case where the content does not match the `Content-Type` (e.g., `application/json`). In the latter case, I would actually expect a `400` response since the request is... well... bad.

Reply:
Yes, it should and yes, it did. Updated.